doc/igprof_linux.md

```diff
@@ -0,0 +1,23 @@
+##Notes for IgProf profiler tool in Linux
+
+Due to various clashes between IgProf and some old Linux versions it is necessary to do some previous configuration in order to run IgProf. It looks like the problem is caused when there is a fork during the execution.
+These are the steps to solve it:
+
+1. Remove the /etc/mtab file
+2. Create a symbolic link in /etc/mtab pointing to /proc/mounts
+3. Execute cvmfs2 with the following options (using the -o flag):
+    * disable_watchdog
+    * fast_parse
+    * uid=0
```

Comment:
Can you check if this is really needed?  It might work just as well without, mounting as a normal user.